angular.json

```diff
@@ -92,7 +92,31 @@
               "fileReplacements": [
                 {
                   "replace": "projects/storefrontapp/src/environments/environment.ts",
-                  "with": "projects/storefrontapp/src/environments/environment.qa.ts"
+                  "with": "projects/storefrontapp/src/environments/environment.ci.2005.ts"
```

Comment:
shouldn't we have this for the cds shellapp as well as it will make it easier for them too

Reply:
also in the package.json

Reply:
CDS should update their own targets. We don't own CDS.